src/components/modules/puzzles/puzzlesBanner/squares/ContentSquares.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import { headline } from '@guardian/src-foundations/typography';
+import { brandAlt, neutral } from '@guardian/src-foundations/palette';
+import { space } from '@guardian/src-foundations';
+import { from, until } from '@guardian/src-foundations/mq';
+import { Square } from './Square';
+import { qrCode } from '../images';
+
+function desktopGridPlacement(row: number, column: number) {
+    return css`
+        ${from.desktop} {
+            grid-row: ${row};
+            grid-column: ${column};
+        }
+    `;
+}
+
+// We can fake the 'gap' rule in IE11 by adding extra rows/columns to act as the gap
+// These gap rows/columns can also be flexible in ways that a standard gap rule can't
+function withIECompatibleGap(rowsOrCols: string[], gap: string) {
+    return rowsOrCols.join(` ${gap} `);
+}
+
+const boxShadow = '0px 6px 0px rgba(0, 0, 0, 0.25);';
```

Comment:
I wonder whether this is something we should suggest to include in Source foundations, so we have a standard box shadow.